src/main/java/Deadline.java

```diff
@@ -0,0 +1,34 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task  {
+    private LocalDate date;
+
+    Deadline(String name, String dateString) {

```

Comment:
did you forget to add in an access modifier here?

Reply:
Why do you say that? Do we need to do so?

Reply:
ah sorry my bad, its just my habit to add access modifiers